packages/fx-core/tests/plugins/resource/function/unit/generateArmTemplates.test.ts

```diff
@@ -46,46 +84,46 @@ describe("FunctionGenerateArmTemplates", () => {
 
     // Assert
     const testProvisionModuleFileName = "functionProvision.result.bicep";
-    const testConfigurationModuleFileName = "functionConfig.result.bicep";
-    const mockedSolutionDataContext = {
-      Plugins: activeResourcePlugins,
-      PluginOutput: {
-        "fx-resource-function": {
-          Provision: {
-            function: {
-              ProvisionPath: `./${testProvisionModuleFileName}`,
-            },
-          },
-          Configuration: {
-            function: {
-              ConfigPath: `./${testConfigurationModuleFileName}`,
-            },
-          },
-          References: {
-            functionAppResourceId: FunctionBicep.functionAppResourceId,
-            functionEndpoint: FunctionBicep.functionEndpoint,
+    const pluginOutput = {
+      "fx-resource-function": {
+        Provision: {
+          function: {
+            ProvisionPath: `./${testProvisionModuleFileName}`,
           },
         },
-        "fx-resource-frontend-hosting": {
-          Outputs: {
-            endpoint: "frontend_hosting_test_endpoint",
-          },
-          References: {
-            domain: "provisionOutputs.frontendHostingOutput.value.domain",
-            endpoint: "provisionOutputs.frontendHostingOutput.value.endpoint",
+        Configuration: {
+          function: {
+            ConfigPath: `./${testConfigurationModuleFileName}`,
           },
         },
-        "fx-resource-identity": {
-          Outputs: {
-            endpoint: "frontend_hosting_test_endpoint",
-          },
-          References: {
-            identityClientId: "provisionOutputs.identityOutput.value.identityClientId",
-            identityResourceId: "userAssignedIdentityProvision.outputs.identityResourceId",
-          },
+        References: {
+          functionAppResourceId: FunctionBicep.functionAppResourceId,
```

Comment:
better to use the raw string as same as identity references. thank you for your update.